src/components/FollowedUsers/FollowedUsers.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import { useGetUserFollows } from '../../actions/useGetUserFollows';
+import { useGetUsers } from '../../actions/useGetUsers';
+import { User } from '../../shared/interfaces/UserInterfaces';
+import { UserCard } from '../UserCard/UserCard';
+import { WithError } from '../WithError/WithError';
+import { WithLoader } from '../WithLoader/WithLoader';
+import './FollowedUsers.scss';
+
+export const FollowedUsers: React.FC = () => {
+  const { userFollows, isLoading: isFollowsLoading, isError: isFollowsError } = useGetUserFollows();
+  const { users, isLoading, isError } = useGetUsers();
+  const [followedUsers, setFollowedUsers] = useState<User[]>();
+
+  useEffect(() => {
+    setFollowedUsers(users.filter(user => userFollows.includes(user.id)));
```

Comment:
I would separate 
users.filter(user => userFollows.includes(user.id))
into another file to make it reusable.

Reply:
I think better way is just helper function in this file, I don't think it should be reausable function becuase filter and includes are enough abstractions for it. It will be too granular rif you do it this way.

Reply:
you mean a separate function in the same file that is called in useEffect ?

Reply:
yes or const at least